scripts/migrate.sh

```diff
@@ -505,26 +508,37 @@ printf "Done."
 wait # wait for subshell to finish
+echo ""
+printf "Update Redis Cache for Console in ${grn}QA${normal}...\n"
+(RADIX_ZONE_ENV="$RADIX_ZONE_ENV" AUTH_PROXY_COMPONENT="$AUTH_PROXY_COMPONENT" CLUSTER_NAME="$DEST_CLUSTER" RADIX_WEB_CONSOLE_ENV="qa" USER_PROMPT="$USER_PROMPT" ./update_redis_cache_for_console.sh)
+wait # wait for subshell to finish
+printf "Update Redis Cache for Console in ${grn}PROD${normal}...\n"
+(RADIX_ZONE_ENV="$RADIX_ZONE_ENV" AUTH_PROXY_COMPONENT="$AUTH_PROXY_COMPONENT" CLUSTER_NAME="$DEST_CLUSTER" RADIX_WEB_CONSOLE_ENV="prod" USER_PROMPT="$USER_PROMPT" ./update_redis_cache_for_console.sh)
+wait # wait for subshell to finish
 
+CUSTOM_INGRESSES=true
 echo ""
-while true; do
-    read -p "Move custom ingresses (e.g. console.*.radix.equinor.com) from source to dest cluster? (Y/n) " yn
-    case $yn in
-        [Yy]* ) source move_custom_ingresses.sh; break;;
-        [Nn]* ) 
-            echo ""
-            echo "Chicken!"
-
-            echo ""
-            echo "For the web console to work we need to apply the secrets for the auth proxy, using the custom ingress as reply url"
-
-            (RADIX_ZONE_ENV="$RADIX_ZONE_ENV" AUTH_PROXY_COMPONENT="$AUTH_PROXY_COMPONENT" WEB_CONSOLE_NAMESPACE="$WEB_CONSOLE_NAMESPACE" AUTH_PROXY_REPLY_PATH="$AUTH_PROXY_REPLY_PATH" ./update_auth_proxy_secret_for_console.sh)
-            wait # wait for subshell to finish
-            (RADIX_ZONE_ENV="$RADIX_ZONE_ENV" AUTH_PROXY_COMPONENT="$AUTH_PROXY_COMPONENT" CLUSTER_NAME="$DEST_CLUSTER" RADIX_WEB_CONSOLE_ENV="qa" USER_PROMPT="$USER_PROMPT" ./update_redis_cache_for_console.sh)
-            wait # wait for subshell to finish
-            (RADIX_ZONE_ENV="$RADIX_ZONE_ENV" AUTH_PROXY_COMPONENT="$AUTH_PROXY_COMPONENT" CLUSTER_NAME="$DEST_CLUSTER" RADIX_WEB_CONSOLE_ENV="prod" USER_PROMPT="$USER_PROMPT" ./update_redis_cache_for_console.sh)
-            wait # wait for subshell to finish
-            break;;
-        * ) echo "Please answer yes or no.";;
-    esac
-done
+if [[ $USER_PROMPT == true ]]; then
+    while true; do
+        read -p "Move custom ingresses (e.g. console.*.radix.equinor.com) from source to dest cluster? (Y/n) " yn
+        case $yn in
+            [Yy]* ) break;;
+            [Nn]* ) CUSTOM_INGRESSES=false; break;;
+            * ) echo "Please answer yes or no.";;
+        esac
+    done
+fi
+
+if [[ $CUSTOM_INGRESSES == true ]]; then
+    source move_custom_ingresses.sh; break;;
+else
+    echo ""
+    echo "Chicken!"
+    echo ""
+    printf "For the web console to work we need to apply the secrets for the auth proxy, using the custom ingress as reply url\n"
+    printf "Update Auth proxy secret...\n"
+    (RADIX_ZONE_ENV="$RADIX_ZONE_ENV" AUTH_PROXY_COMPONENT="$AUTH_PROXY_COMPONENT" WEB_CONSOLE_NAMESPACE="$WEB_CONSOLE_NAMESPACE" AUTH_PROXY_REPLY_PATH="$AUTH_PROXY_REPLY_PATH" ./update_auth_proxy_secret_for_console.sh)
+    wait # wait for subshell to finish
+fi
+printf "\n"
+printf "${grn}Done.${normal}\n"
```

Comment:
Add newline at end of file
```suggestion
printf "${grn}Done.${normal}\n"

```